extensions/linear/package.json

```diff
@@ -123,6 +123,15 @@
       ]
     }
   ],
+  "preferences": [
```

Comment:
Would it be better to add the preference only to the relevant commands instead of a global one for the extension?

Reply:
Since it's an issue-related action, it means the preference is used whenever a command shows an issue list/detail which is the case for Active Cycle, Assigned Issues, Created Issues, Search Issues, Search Projects (project issues), Notifications (when you open the related issue).

In my opinion, users would rather want to customize how they copy and send the issue to teammates on a global level than personalize it for 6 different commands. What do you think?

Reply:
Yeah, makes sense.